### DISCUSSION TITLE
Good review and thanks

### DISCUSSION THREAD

**Discussion by dgnuff** [valinet/ExplorerPatcher#4031] [General]
Just to say that ExplorerPatcher may have saved my sanity. :)  Within a few months of starting to use Windows 95, I got into the habit of placing the taskbar on the right side of the screen.

I'd read the reports that the old Windows 10 taskbar was finally being removed from Windows 11 24H2, and I feared the worst.

But I'm glad to find that the Windows 10 style taskbar that's built into ExplorerPatcher can be dragged to the right, thus solving the problem for me.

And the icing on the cake is that ExplorerPatcher co-operates flawlessly with the other Shell modification program I use: Open Shell.

**Comment by imonxda2** [comment]
Hi, I was looking for a good place to voice my DEEP appreciation for this project. I think this is a good spot. So, THANK YOU SOOOOOOOOO MUCH FOR THIS!!!!!

**Comment by jeffkshaw** [comment]
I love explorerpatcher.  Thanks for your efforts.

**Comment by ebuck79** [comment]
Can you add the charms bar back please in the next update
